layouts/partials/header.html

```diff
@@ -1,5 +1,8 @@
 <div id="header">
     <h1><a href="{{ .Site.BaseURL }}" id="logo">
+        {{- if .Site.Data.globalheader.image }}
+        <img src="{{ .Site.Data.globalheader.image | relURL }}" alt="{{ .Site.Data.globalheader.image_alt }}" />
+        {{- end }}
         {{ with .Site.Data.globalheader.title }}{{ . | markdownify }}{{ else }}{{ .Site.Title }}{{ end }}
```

Comment:
I wonder if the `else` here should be removed so it's easy to use an image-only header... Would break back-compat though :thinking:

Reply:
Good idea. Another way could be something like:

```
{{ with .Site.Data.globalheader }}
    image and title content without else
{{ else }}
    {{ .Site.Title }}
{{ end }}
```